src/features/tutorials.tsx

```diff
@@ -1246,7 +1229,6 @@ export const setVersion = (v: string) => {
   version = v;
 };
 
-const workbenchCommands = new Set<() => void>();
 let topbarObserver: MutationObserver;
 export let enabled = false;
 export const toggleFeature = (flag: boolean) => {
```

Comment:
instead of exporting everything out to `src/index.ts`, let's pass in `extensionAPI` into `toggleFeature` as that will limit the refactor. This also allows us to remove when a feature is toggled off

Reply:
I'm pretty blocked on this.

Normally I would think to `import` from a defined `export` somewhere, but it seems `extensionAPI` is in a return as a parameter of `onload`, and I'm not sure what to do with that.

Would you be able to get on a call?

Reply:
I think I got it.